store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreConstants.java

```diff
@@ -18,8 +18,17 @@
 
 public final class FederatedStoreConstants {
     // Operation options
-    public static final String GRAPH_IDS = "gaffer.federatedstore.operation.graphIds";
-    public static final String SKIP_FAILED_FEDERATED_STORE_EXECUTE = "gaffer.federatedstore.operation.skipFailedFederatedStoreExecute";
+    public static final String GAFFER_FEDERATED_STORE = "gaffer.federatedstore";
+    public static final String GRAPH_IDS = "graphIds";
+    public static final String KEY_OPERATION_OPTIONS_GRAPH_IDS = GAFFER_FEDERATED_STORE + "operation.graphIds";
+    public static final String KEY_GRAPH_IDS = GAFFER_FEDERATED_STORE + "." + GRAPH_IDS;
+    public static final String KEY_SKIP_FAILED_FEDERATED_STORE_EXECUTE = "gaffer.federatedstore.operation.skipFailedFederatedStoreExecute";
+    public static final String FILE = "file";
+    public static final String SCHEMA = "schema";
+    public static final String PROPERTIES = "properties";
+    public static final String ID = "id";
+    public static final String CUSTOM_PROPERTIES_AUTHS = "customPropertiesAuths";
+    public static final String AUTHS = "auths";
```

Comment:
Please could you have some javadoc for these constants. I don't think they are all "Operation options".

Reply:
👍